src/devices/tuya.ts

```diff
@@ -19136,7 +19136,155 @@ export const definitions: DefinitionWithExtend[] = [
+            e.child_lock(),
+            e
+                .climate()
+                .withSetpoint("current_heating_setpoint", 5, 35, 0.5, ea.STATE_SET)
+                .withLocalTemperature(ea.STATE)
+                .withLocalTemperatureCalibration(-9, 9, 1, ea.STATE_SET)
+                .withSystemMode(["off", "heat"], ea.STATE)
+                .withPreset(["comfort", "eco", "antifrost", "off", "comfort_1", "comfort_2", "program", "manual"]),
+            e
+                .enum("mode", ea.STATE, ["comfort", "eco", "antifrost", "off", "comfort_1", "comfort_2", "program", "manual"])
+                .withDescription("Current running mode")
+                .withCategory("diagnostic"),
+            e
+                .numeric("comfort_temperature", ea.STATE_SET)
+                .withUnit("  C")
+                .withValueMin(5)
+                .withValueMax(35)
+                .withValueStep(0.5)
+                .withDescription("Set comfort temperature")
+                .withCategory("config"),
+            e
+                .numeric("eco_temperature", ea.STATE_SET)
+                .withUnit("  C")
+                .withValueMin(5)
+                .withValueMax(30)
+                .withValueStep(0.5)
+                .withDescription("Set ECO temperature")
+                .withCategory("config"),
+            e
+                .numeric("antifrost_temperature", ea.STATE_SET)
+                .withUnit("  C")
+                .withValueMin(5)
+                .withValueMax(15)
+                .withValueStep(0.5)
+                .withDescription("Set antifrost temperature")
+                .withCategory("config"),
+            e
+                .numeric("temperature_sensibility", ea.STATE_SET)
+                .withUnit("  C")
+                .withValueMin(0.5)
+                .withValueMax(5)
+                .withValueStep(0.5)
+                .withDescription("Set thermostat sensitivity")
+                .withCategory("config"),
+            e.binary("antifrost", ea.STATE_SET, "ON", "OFF").withDescription("Enable antifrost protection feature").withCategory("config"),
+            e
+                .binary("window_detection", ea.STATE_SET, "ON", "OFF")
+                .withLabel("Open window detection")
+                .withDescription("Enable / Disable open window detection feature")
+                .withCategory("config"),
+            e.enum("window", ea.STATE, ["close", "open"]).withDescription("Indicates if window is open").withCategory("diagnostic"),
+            e.numeric("power", ea.STATE).withUnit("W").withDescription("Instantaneous measured power").withCategory("diagnostic"),
+            e.numeric("voltage", ea.STATE).withUnit("V").withDescription("Measured electrical potential value").withCategory("diagnostic"),
+            e.numeric("current", ea.STATE).withUnit("mA").withDescription("Instantaneous measured electrical current").withCategory("diagnostic"),
```

Comment:
```suggestion
            e.current(),
```

Note that the unit here is A and not mA, updated the `tuyaDatapoints` converter accordingly.

Reply:
Hi, did see the update of the tuyaDatapoints after changing from mA to A ? Need to divide by 1000 the datapoint ?